src/ext/NetFx/wixlib/NetFxExtension_Platform.wxi

```diff
@@ -5,23 +5,32 @@
   <?include ..\..\caDecor.wxi ?>
 
   <Fragment>
-    <CustomAction Id="$(var.Prefix)NetFxScheduleNativeImage$(var.Suffix)" DllEntry="SchedNetFx" Execute="immediate" Return="check" SuppressModularization="yes" BinaryRef="$(var.Prefix)NetFxCA$(var.Suffix)" />
+    <CustomAction Id="$(var.Prefix)NetFxScheduleNativeImage$(var.Suffix)" DllEntry="SchedNetFx" Execute="immediate" Return="ignore" SuppressModularization="yes" BinaryRef="$(var.Prefix)NetFxCA$(var.Suffix)" />
```

Comment:
Why change this to `ignore`?

Reply:
Previously, in NetFx extension, `NativeImage` was the only custom element. Its custom action fails if a view cannot be opened on `Wix4NetFxNativeImage` table. Now, there is a new custom element `DotNetCompatibilityCheck` whose custom action also fails if a view cannot be opened on its own table `Wix4NetFxDotNetCompatibilityCheck`. Using just one of the two elements would result with the other one's CA failing thus failing the entire installation, so using `ignore` for both made sense.

Another possibility is to make unsuccessful calls to `WcaOpenExecuteView` an early exit condition for the CA without failing it.

Reply:
You'll want to use `WcaTableExists()` from `wcautil.h` to branch around these two code paths, instead.

Reply:
I stumbled upon a limitation when using `WcaTableExists()`. If table name is 32 characters or longer, it always returns `E_FAIL` because, as it seems, `::MsiDatabaseGetPrimaryKeysW()` cannot handle long names. The fix was easy - to shorten the table name, I just thought the limitation was worth noting.